Regression/WarpX-tests.ini

```
@@ -4806,3 +4806,20 @@ doVis = 0
 compareParticles = 0
 doComparison = 0
 analysisRoutine = Examples/Tests/openbc_poisson_solver/analysis.py
+
+[ThomsonParabolaSpectrometer]
+buildDir = .
+inputFile = Examples/Physics_applications/thomson_parabola_spectrometer/inputs
+runtime_params = warpx.abort_on_warning_threshold=high
```

```suggestion
runtime_params = warpx.abort_on_warning_threshold=low
```

The test does not seem to raise any warning at the moment. We can lower the threshold to abort to `low`, I think.